rest/route/github.go

```diff
@@ -306,6 +299,60 @@ func (gh *githubHookApi) getCommentLogWithMessage(event *github.IssueCommentEven
 	}
 }
 
+func (gh *githubHookApi) displayHelpText(ctx context.Context, owner, repo string, prNum int) error {
+	pr, err := gh.sc.GetGitHubPR(ctx, owner, repo, prNum)
+	if err != nil {
+		return errors.Wrap(err, "getting PR from GitHub API")
+	}
+
+	if pr == nil || pr.Base == nil || pr.Base.Ref == nil {
+		return errors.New("PR contains no base branch label")
+	}
+	branch := pr.Base.GetRef()
+	projectRefs, err := model.FindMergedEnabledProjectRefsByRepoAndBranch(owner, repo, branch)
+	if err != nil {
+		return errors.Wrapf(err, "fetching project ref for repo '%s/%s' with branch '%s'",
+			owner, repo, branch)
+	}
+	helpMarkdown := getHelpTextFromProjects(projectRefs)
+	return gh.sc.AddCommentToPR(ctx, owner, repo, helpMarkdown, prNum)
+}
+
+func getHelpTextFromProjects(projectRefs []model.ProjectRef) string {
+	var manualPRProjectEnabled, autoPRProjectEnabled, cqProjectEnabled bool
+	var cqProjectMessage string
+
+	for _, p := range projectRefs {
+		if p.IsManualPRTestingEnabled() {
+			manualPRProjectEnabled = true
+		}
+		if p.IsAutoPRTestingEnabled() {
+			autoPRProjectEnabled = true
+		}
+		if err := p.CommitQueueIsOn(); err == nil {
+			cqProjectEnabled = true
+		} else if cqMessage := p.CommitQueue.Message; cqMessage != "" {
+			cqProjectMessage += cqMessage
+		}
+	}
+
+	formatStr := "- `%s` \n    - %s\n"
+	res := fmt.Sprintf("### %s\n", "Available Evergreen Comment Commands")
+	if autoPRProjectEnabled || manualPRProjectEnabled {
+		comments := fmt.Sprintf("%s, %s", retryComment, patchComment)
+		res += fmt.Sprintf(formatStr, comments, "attempt creating PR patch")
```

Comment:
nit: this has a different tense than the rest of them. Should this be `"attempts to create PR patch"` to be consistent with the rest of them?

Maybe should also note that it's a new PR patch to clarify that it's not just restarting an existing one?